easybuild/easyconfigs/g/gmsh/gmsh-3.0.6-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'CMakeMake'
+
+name = 'gmsh'
+version = '3.0.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://geuz.org/gmsh'
+description = """Gmsh is a 3D finite element grid generator with a build-in CAD engine and post-processor."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {"usempi": True}
+
+source_urls = ['http://gmsh.info/src/']
+sources = ['%(name)s-%(version)s-source.tgz']
+checksums = ['9700bcc440d7a6b16a49cbfcdcdc31db33efe60e1f5113774316b6fa4186987b']
+
+builddependencies = [
+    ('CMake', '3.10.2'),
+    ('SWIG', '3.0.12', versionsuffix),
+]
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('PETSc', '3.8.3', '-downloaded-deps'),
+    ('SLEPc', '3.8.3', versionsuffix),
```

Comment:
@wpoely86 To use `SLEPc` from #6260, you'll need to drop the `versionsuffix` here...